build/makelib/common.mk

```diff
@@ -31,6 +31,10 @@ DOCKERCMD=$(shell podman version >/dev/null 2>&1 && echo podman)
 endif
 endif
 
+ifeq ($(DOCKERCMD),)
+    $(error "No running container engine found.")
```

Comment:
This error is shown in the [mac CI](https://github.com/rook/rook/actions/runs/7328295460/job/19956369026?pr=13474). However, I don't get that error locally with this change. Not sure why the CI is different.

Reply:
Yeah, that mac CI failed. I should dig how these makefile work with mac.

Reply:
The CI always uses `docker`, so maybe just add an env var to set `DOCKERCMD=docker` in `.github/workflows/build.yml` under the mac action